core/java/android/provider/Settings.java

```diff
@@ -8323,6 +8323,12 @@ public boolean validate(@Nullable String value) {
          */
         public static final String FLASHLIGHT_ENABLED = "flashlight_enabled";
 
+        /**
+         * Whether or not fingerprint unlock is allowed on Keyguard.
+         * @hide
+         */
+        public static final String FINGERPRINT_UNLOCK_KEYGUARD_ENABLED = "fingerprint_unlock_keyguard_enabled";
```

Comment:
I realized this file contains a `SETTINGS_TO_BACKUP` array; should I put this new setting into that array?

Reply:
I'm not sure.